### DISCUSSION TITLE
Could not create new data base.

### DISCUSSION THREAD

**Discussion by Aafiq2004** [pawelsalawa/sqlitestudio#4938] [General]

**Comment by pawelsalawa** [comment]
What exactly are you doing and what error are you getting?

**Reply by asauppe** [reply]
I am having the same issue - there is no button like there used to be in the interface. Whereas "Add a Database" used to give the option of creating a new database or adding a currently existing database file into the SQLite interface, now only the latter option is available.

<img width="512" alt="Screenshot 2024-03-11 at 11 11 50 AM" src="https://github.com/pawelsalawa/sqlitestudio/assets/6502546/6e950265-dc9b-4aaf-b6e2-aae16a80bb19">

I ended up using the command line to touch a file with the desired name/extension, and then adding the database through the UI and creating as usual. But this is a less than desirable workaround when the point of the UI is to alleviate the need for working on the console with SQLite.

**Reply by pawelsalawa** [reply]
Is this under Ubuntu, or MacOS?

**Comment by asauppe** [comment]
MacOS.
________________________________

**Reply by pawelsalawa** [reply]
Have you tried creating new database using this button that you see? Recent versions use just this single button to both create new databases and open existing ones.

**Comment by asauppe** [comment]
Yes, this is the only option, to load a file:


________________________________

**Reply by pawelsalawa** [reply]
So, did you click that button and you were unable to enter new file to create?

**Comment by asauppe** [comment]
Correct. I can select a folder in there, then enter a new database name in the previous window, but you can then see the error in SQLite Studio when I try to create. The only thing that has worked has been to touch a file on the command line, and then open/load that as a blank file, and create tables from there.


________________________________

**Reply by pawelsalawa** [reply]
> but you can then see the error in SQLite Studio when I try to create

What does the error say?

**Comment by asauppe** [comment]
It is in the console of the second screenshot: "Could not add database <path>: Could not open database: unable to open database file."
________________________________

**Comment by asauppe** [comment]
I'm realizing my screenshots aren't here. See below:

<img width="912" alt="Screenshot 2024-03-12 at 6 58 13 PM" src="https://github.com/pawelsalawa/sqlitestudio/assets/6502546/759cf8eb-3b63-4b3f-862e-b37fe5fa6387">
<img width="512" alt="Screenshot 2024-03-12 at 7 14 39 PM" src="https://github.com/pawelsalawa/sqlitestudio/assets/6502546/d41351bf-450c-4d13-ad20-2efcdd9ab1e9">
<img width="1032" alt="Screenshot 2024-03-12 at 7 14 45 PM" src="https://github.com/pawelsalawa/sqlitestudio/assets/6502546/2eeb13ec-137c-4efc-acd1-c6cbea7b2fa8">

**Reply by pawelsalawa** [reply]
Okay, it seems this is already reported as #4796
